lib/examples/gauge/svg.ex

```diff
@@ -42,6 +42,7 @@ defmodule Examples.Gauge.Svg do
           d="<%= @settings.d_gauge_half_circle %>">
         </path>
         <path id="gauge-value"
+          <%= if String.length(@settings.d_value_color) != 0 do %> style="stroke: <%= @settings.d_value_color %>;" <% end %>
```

Comment:
Maybe it would be better pass here the `class` (where the color would be defined) instead of inline `style`...